org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -297,6 +327,11 @@ class ${relation.Name}(Relation):
     #end
 #end
 
+# --- References to States  ---
+#foreach($value in $allStates.values())
```

Comment:
I tried the bitcounter example, and nothing showed up here, can you check why? It should declare all the 7 global variables (eg ref_counter, ref_counter_bit2, ref_counter_bit2_bit22) here. I am surprised that it worked without these global variable declarations.
![image](https://user-images.githubusercontent.com/40646603/218631638-966aff98-6825-4b51-a724-26866b683746.png)

Reply:
I replaced "$allStates.values()" with "$allStates". It's working now.